libpod/container_top_linux.c

```diff
@@ -1,5 +1,5 @@
 //go:build !remote
-// +build !remote
+
```

Comment:
nit: unnecessary newline?

Reply:
Should I create another PR to remove this unnecessary newline?

Reply:
Sure although not critical